example/lib/main.dart

```diff
@@ -8,7 +8,6 @@ void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   Stripe.publishableKey = stripePublishableKey;
   Stripe.merchantIdentifier = 'merchant.flutter.stripe.test';
-  Stripe.urlScheme = 'flutterstripe';
```

Comment:
done on purpose?

Reply:
Yes, url scheme should match a custom scheme. On web it is used to redirect back after a payment has been successful and defaults to the window url